e2e/cypress/e2e/issue_workflow_test/DPP/issueDPP.cy.ts

```diff
@@ -12,13 +12,16 @@ class DPPIssueFlow extends IssuePage {
   }
 
   testUNTPTestSuite() {
-    this.logCurrentDir();
-    this.runShellScript('./cypress/e2e/issue_workflow_test/DPP/test-untp-dpp-scripts.sh');
-    this.deleteFile('DigitalProductPassport_instance-v0.5.0.json');
+    this.runUntpTest('digitalProductPassport', 'v0.5.0');
   }
 }
 
 describe('Issue DPP end-to-end testing flow', () => {
+  before(() => {
+    // Clear the lastCredential from Cypress environment after each test
+    Cypress.env('lastCredential', undefined);
+  });
+
```

Comment:
It looks like we consistently clear the last credential before every single test? Could we instead run this in a `support/e2e.ts` or similar  `before()` call, or do you think we might want some tests which don't clear the credential?

See https://docs.cypress.io/app/core-concepts/writing-and-organizing-tests#Support-file , especially the section with the
```javascript
  cy.log('I run before every test in every spec file!!!!!!')
```

Reply:
No problem. Sorry, I meant to approve earlier either way. Doing so now.